src/core/global.ts

```diff
@@ -6,9 +6,9 @@
  * @hidden
  * @private
  */
-declare var global: any;
+declare var globalThis: any;
 /**
  * @hidden
  */
-const toExport: any = typeof(window) !== 'undefined' ? window : typeof(self) !== 'undefined' ? self : global;
+const toExport: any = typeof(window) !== 'undefined' ? window : typeof(self) !== 'undefined' ? self : typeof (global) !== "undefined" ? global : globalThis;
```

Comment:
@pkit 

Could we change this to simply use `globalThis` since it supported across all major browsers (including Node)?

e.g. `const toExport: any = globalThis;`